config/development_archival.yaml

```diff
@@ -0,0 +1,127 @@
+persistence:
+  defaultStore: cass-default
+  visibilityStore: cass-visibility
+  numHistoryShards: 4
+  datastores:
+    cass-default:
+      cassandra:
+        hosts: "127.0.0.1"
+        keyspace: "temporal"
+        user: "mark"
+        password: "password"
+    cass-visibility:
+      cassandra:
+        hosts: "127.0.0.1"
+        keyspace: "temporal_visibility"
+global:
+  membership:
+    name: temporal
+    maxJoinDuration: 30s
+    broadcastAddress: "127.0.0.1"
+  pprof:
+    port: 7936
+
+services:
+  frontend:
+    rpc:
+      grpcPort: 7233
+      membershipPort: 6933
+      bindOnLocalHost: true
+    metrics:
+      statsd:
+        hostPort: "127.0.0.1:8125"
+        prefix: "temporal"
+
+  matching:
+    rpc:
+      grpcPort: 7235
+      membershipPort: 6935
+      bindOnLocalHost: true
+    metrics:
+      statsd:
+        hostPort: "127.0.0.1:8125"
+        prefix: "temporal"
+
+  history:
+    rpc:
+      grpcPort: 7234
+      membershipPort: 6934
+      bindOnLocalHost: true
+    metrics:
+      statsd:
+        hostPort: "127.0.0.1:8125"
+        prefix: "temporal"
+
+  worker:
+    rpc:
+      grpcPort: 7239
+      membershipPort: 6939
+      bindOnLocalHost: true
+    metrics:
+      statsd:
+        hostPort: "127.0.0.1:8125"
+        prefix: "temporal"
+
+clusterMetadata:
+  enableGlobalNamespace: false
+  failoverVersionIncrement: 10
+  masterClusterName: "active"
+  currentClusterName: "active"
+  clusterInformation:
+    active:
+      enabled: true
+      initialFailoverVersion: 0
+      rpcName: "frontend"
+      rpcAddress: "localhost:7933"
+
+dcRedirectionPolicy:
+  policy: "noop"
+  toDC: ""
+
+archival:
```

Comment:
just curious – were you able to start temporal server using this config file? if not, i wonder if it is worth it to postpone adding `development_archival.yaml` until we release tested/supported archival feature and have a chance to really test this.

Reply:
@markmark206 this configuration is the same as the previous `development.yaml` which had archival enabled. Should we do a separate testing of it?

Reply:
@markmark206 this configuration is the same as the previous development.yaml which had archival enabled and that we've been running. Should we do a separate testing of it?

Reply:
we talked about testing all config files at some point, but until then, just running `./temporal-server -e development_archival start` would be enough. (since this is a copy paste of another file, it's likely it just works, and it's not breaking any workflows, so i think this is also fine as is.)

Reply:
tested and i can see the default archival state set to Enabled if started using development_archival config